python/pyspark/ml/evaluation.py

```diff
@@ -193,9 +193,7 @@ class RegressionEvaluator(JavaEvaluator, HasLabelCol, HasPredictionCol):
 
     .. versionadded:: 1.4.0
     """
-    # Because we will maximize evaluation value (ref: `CrossValidator`),
-    # when we evaluate a metric that is needed to minimize (e.g., `"rmse"`, `"mse"`, `"mae"`),
-    # we take and output the negative of this metric.
```

Comment:
In Spark 1.4, we negated some metrics from ```RegressionEvaluator``` since ```CrossValidator``` always maximized metrics. This was fixed in 1.5, #12377 updated the Scala docs, this PR updates the Python docs.